changelog.d/105.misc

```diff
@@ -0,0 +1 @@
+Ensure the Rust reporter passes type checking with jaeger-client 4.7's type annotations.
```

Comment:
I'm guessing you leave this so when you do a merge it will properly get removed?

Reply:
It would fail the newsfile check though.
In general on this fork we don't care that much about changelog entries besides making them pass CI because we don't do proper releases.